CoreScriptsRoot/Modules/LoadingScreen3D.lua

```diff
@@ -0,0 +1,355 @@
+-- LoadingScreen3D.lua --
+-- Written by Kip Turner, copyright ROBLOX 2016 --
+
+local GUI_DISTANCE_FROM_CAMERA = 6
+local VERTICAL_SCREEN_PERCENT = 1/3
+local HORIZONTAL_SCREEN_PERCENT = 1/3
+local SECOND_TO_FADE = 2.5
+local ROTATIONS_PER_SECOND = 0.5
+local TEXT_SCROLL_SPEED = 25
+
+local CoreGui = game:GetService('CoreGui')
+local RunService = game:GetService('RunService')
+local MarketPlaceService = game:GetService('MarketplaceService')
+local UserInputService = game:GetService('UserInputService')
+local ReplicatedFirst = game:GetService('ReplicatedFirst')
+local GuiService = game:GetService('GuiService')
+
+local RobloxGui = CoreGui:WaitForChild("RobloxGui")
+local Util = require(RobloxGui.Modules.Settings.Utility)
+
+
+local function FadeElements(element, newValue, tweeners, duration)
```

Comment:
whats the point of the tweeners? seems we always pass an empty table that has no reference